HomeWork_11/HW11.py

```diff
@@ -0,0 +1,74 @@
+from datetime import datetime
+from time import perf_counter
+
+
+
+def decorator(func):
+    def timer():
+        start = perf_counter()
+        def inner():
+            return perf_counter() - start
+        return inner
+
+def counter(func):
+    count = 0
+    def inner(*args,**kwargs):
+        nonlocal count
+        count +=1
+        print(f"Функція {func.__name__} викликалась {count} разів")
+        return func(*args,**kwargs)
+
+
+@counter
+@decorator
+def add(a, b):
+    a = int(input("first digit "))
+    b = int(input("twice digit "))
+    return a + b
+
+
+
+
+# #-----------------------------------------
+def MyCustomException(Exception):
+    pass
+try:
+    a,b/0
+except:
+    MyCustomException
+    print("Custom exception is occured")
+
+#==========================
+class FileManager():
```

Comment:
Your custom context manager doesn't need to be an implementation of the `open` function. According to the task, your context manager should show where it stars (print 10 =), print error if there is one, and show were it ends (print 10 =)

So, the simple implementation can be something like this:
```
class MyContext:
    def __enter__(self):
        print('==========')

    def __exit__(self, exc_type, exc_val, exc_tb):
        if exc_val:
            print(exc_val)
        print('==========')
        return True
````